window/window.go

```diff
@@ -50,7 +55,7 @@ func Hamming(L int) []float64 {
 		N := L - 1
 		coef := math.Pi * 2 / float64(N)
 		for n := 0; n <= N; n++ {
-			r[n] = 0.54 - 0.46*math.Cos(coef*float64(n))
```

Comment:
I don't think these need to change. The mathworks site for FlatTop specifically lists a table of aN constants which is why those should be there.